selftests/functional/test_replay_basic.py

```diff
@@ -58,9 +58,9 @@ def test_run_replay_latest(self):
         self.run_and_check(cmd_line, expected_rc)
 
     def test_run_replay_data(self):
-        file_list = ['multiplex', 'config', 'urls', 'pwd', 'args']
+        file_list = ['multiplex', 'config', 'urls', 'pwd', 'args', 'cmdline']
         for filename in file_list:
-            path = os.path.join(self.jobdir, 'replay', filename)
+            path = os.path.join(self.jobdir, 'jobdata', filename)
```

Comment:
OK, but this adds a fallback feature, which ought to be tested. I'd suggest copying at least a simple replay check, run the job, renamed the `jobdata` to `replay` and then run the test on that directory.